Source/Internal/IGListSectionMap.h

```diff
@@ -38,7 +38,7 @@ IGLK_SUBCLASSING_RESTRICTED
  @param objects The objects in the collection.
  @param sectionControllers The section controllers that map to each object.
  */
-- (void)updateWithObjects:(NSArray <id <NSObject>> *)objects sectionControllers:(NSArray <id <NSObject>> *)sectionControllers;
+- (void)updateWithObjects:(NSArray <id <NSObject>> *)objects sectionControllers:(NSArray <IGListSectionController *> *)sectionControllers;
```

Comment:
at some point this method was not updated to the `IGListSectionController<IGListSectionType> *` type and was missed in a few refactorings. the next method down, `sectionControllerForSection:` returned an `IGListSectionController *` so the types were a significant mismatch and needed to be updated.